test/spec/modules/tappxBidAdapter_spec.js

```diff
@@ -293,4 +295,39 @@ describe('Tappx bid adapter', function () {
       expect(consent[0].url).to.match(/&type=iframe/);
     });
   })
+
+  describe('module Floor implementation', function() {
+    let bidderRequest_f = c_BIDREQUEST;
```

Comment:
set your vars like so

```
  let bidderRequest_f,
    getFloorResponse;
```

Then make them whatever your default assignment is inside a beforeEach block like so:

https://github.com/prebid/Prebid.js/blob/master/test/spec/modules/rubiconBidAdapter_spec.js#L20-L24

https://github.com/prebid/Prebid.js/blob/master/test/spec/modules/rubiconBidAdapter_spec.js#L300-L301

This helps stablilize tests if something messes with these vars or tests go out of order for some reason.

Reply:
Change of variables getFloorResponse and bidderRequest_f and added the beforeEach block.
: )